hw/drivers/sensors/bma253/src/bma253_shell.c

```diff
@@ -256,10 +271,11 @@ bma253_stream_read_cmd(struct bma253 * bma253,
         return EINVAL;
     }
 
-    return bma253_stream_read(bma253,
-                              bma253_stream_read_cb,
-                              &ctx,
-                              0);
+    return bma253_stream_read(&bma253->sensor,   //zg
+            SENSOR_TYPE_ACCELEROMETER,   //zg
```

Comment:
As per the mynewt coding style guide, we should not be using C++ comments. Can you please change it to C style comments. `/*    */`